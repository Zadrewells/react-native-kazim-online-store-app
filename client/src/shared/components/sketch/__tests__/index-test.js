import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {
	Sketch
} from 'src/shared/components';

jest.useFakeTimers();
Date.now = jest.fn(() => 1503187200000);

describe('<Sketch />', () => {
	const getWrapper = props => renderer.create(
		<Sketch
			{
			...props
			}
		/>
	).toJSON();

	const modelI = {
		width: 100,
		heigth: 100
	};

	it('renders correctly', () => {
		const tree = getWrapper(modelI);
		expect(tree).toMatchSnapshot();
	});
});
