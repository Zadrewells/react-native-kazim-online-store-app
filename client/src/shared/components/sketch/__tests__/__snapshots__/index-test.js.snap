// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sketch /> renders correctly 1`] = `
<View
  height={50}
  style={
    Object {
      "backgroundColor": "#e4e2e2",
      "height": 50,
      "opacity": 0.5,
      "width": 100,
    }
  }
  width={100}
/>
`;
